### ISSUE TITLE
Stack over flow for large corpora

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mivoq/hunpos#19]
```
I am trying to make a new model with hunpos but it has issues with large 
corpora. My corpus is 300 000 words and I get stack overflow problem.
I am using it on linux. Does this POS tagger support larger corpora? Or this 
problem is because of my OS and my Computer RAM? 

```

Original issue reported on code.google.com by `Shadi...@gmail.com` on 19 Feb 2014 at 9:50

**Comment by giuliopaci** [comment]
In the clib part of the code there still is a stack overflow that fails even if there is an empty line every ten tokens.

**Comment by giuliopaci** [comment]
The issue in clib has been solved in revision 307da3771fe87fd367f1f1c2243c6708b9091706.
